### PR TITLE
Adds valid PAT testing to existing ec2 pipes

### PR DESCRIPTION
**Pull request by supertopher**
adds the same workflow seen in gradle.yml to other pipelines

## What
grade.yml is running through PAT keys to check for valid keys.  lets do that elsewhere

## How
adds a need before GitHub token invoking the supertopher branch

### CODE REVIEW

File: .github/workflows/gke-kube-test-command.yml
Comment (davinchia) [inline .github/workflows/gke-kube-test-command.yml:31]: we should change these before merging this in.

Comment (davinchia) [reply .github/workflows/gke-kube-test-command.yml:31]: same for the rest of the jobs.

**Review by davinchia — APPROVED** [review]
Nice work! Fingers crossed!

File: .github/workflows/github-community-label-bot.yaml
Comment (supertopher) [inline .github/workflows/github-community-label-bot.yaml:15]: Whatever we were using before it's Octavia now, I'm killing these secrets after this

**Review by supertopher — COMMENTED** [review]
LGTM assuming testing passes I'll merge away

**Pull request merged**
